js/index.js

```diff
@@ -40,3 +40,119 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+const imgID =
+document.getElementById("logo-img")
+// console.log(imgID)
+
+const Idimg =
+document.getElementById("cta-img")
+// console.log(Idimg)
+
+const navStyles = 
+document.querySelectorAll("nav a")
+
+navStyles.forEach(element => {
+  element.stlye.color = 'green'
+});
+
+const newCont = 
+document.querySelector('header')
+
+// nav
+
+const NewNav =
```

Comment:
Creating a new elements takes in 3 steps:

1. Create an element using `newElement = document.createElement("")`, passing whatever element you want to create inside of the parenthesis inbetween to quotes

2. add content to the newly created element

3. prepend or appendChild the newly created array using this syntax: 
```
whereToPlace.prepend(newElement)
```